google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractStructReader.java

```diff
@@ -259,6 +265,19 @@ public String getPgJsonb(String columnName) {
     return getPgJsonbInternal(columnIndex);
   }
 
+  @Override
+  public long getPgOid(int columnIndex) {
+    checkNonNullOfType(columnIndex, Type.pgOid(), columnIndex);
+    return getPgOidInternal(columnIndex);
+  }
+
+  @Override
+  public long getPgOid(String columnName) {
+    int columnIndex = getColumnIndex(columnName);
+    checkNonNullOfType(columnIndex, Type.pgOid(), columnName);
+    return getPgOidInternal(columnIndex);
+  }
```

Comment:
Would it be worth considering not adding any of this at all, and instead instruct users that they have to get PG.OID values in the same way as INT64? So just call getLong(..) and the other corresponding methods. This is otherwise all just a copy-paste of those methods, and there's no additional validation of the values that we are getting.
I know that it is not what we have done for other types (e.g. we could in theory have done the same for JSON by telling users to just call getString(...)). But this might be the right time to start to prevent us from doing double work for future types, while it does not add any additional type safety or features for users.

@skuruppu Any thoughts on this?

Reply:
Done.
I had an offline chat with @skuruppu and treating OID as Long is actually what we are doing for the other clients. I have removed all external PG.OID getters.